src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -951,7 +951,7 @@ static bool elem_consistent_with_arr(BasicType elem_bt, const TypeAryPtr* arr_ty
 //  S extends VectorSpecies<E>>
 // VM load(Class<? extends VM> vmClass, Class<E> elementType, int length,
 //         Object base, long offset,    // Unsafe addressing
-//         C container, long index, S s,     // Arguments for default implementation
+//         C container, long index, S s,  boolean from_ms,    // Arguments for default implementation
```

Comment:
It's misleading to place the new argument at the end because it's not part of the default implementation. Place it just after the unsafe addressing part? Also i recommend renaming it to `fromSegment` here and in the Java code (but no need in the C++ code).